packages/next-swc/crates/next-core/js/src/entry/next-hydrate.tsx

```diff
@@ -33,7 +33,6 @@ async function loadPageChunk(assetPrefix: string, chunkData: ChunkData) {
 
   window.next = {
     version: version || '',
-    // @ts-expect-error
```

Comment:
I started getting a TS error from this, saying there was no error there. I think our types might be busted in this package. In any case, I tried to improve our typing a bit (see above).